packages/mdx/lib/core.js

```diff
@@ -103,7 +107,15 @@ export function createProcessor(options = {}) {
   pipeline
     .use(remarkMarkAndUnravel)
     .use(remarkPlugins || [])
-    .use(remarkRehype, {allowDangerousHtml: true, passThrough: nodeTypes})
+    .use(remarkRehype, {
+      ...remarkRehypeOptions,
+      allowDangerousHtml: true,
+      passThrough: Array.isArray(remarkRehypeOptions.passThrough)
+        ? Array.from(
+            new Set([...remarkRehypeOptions.passThrough, ...nodeTypes])
+          )
+        : nodeTypes
```

Comment:
```suggestion
      passThrough: [...(remarkRehypeOptions.passThrough || []), ...nodeTypes]
```

1. No need to be unique
2. No need to explicitly check for an array, it’s supposed to be an array or missing

This branch should also be covered by the tests. I’m not sure of a practical and useful thing to test though!

Reply:
yep, i was thinking about this as well and couldn't really come up with something.

Reply:
Can you add a C8 ignore comment then?